packages/editor-ui/src/stores/settings.store.ts

```diff
@@ -173,6 +174,12 @@ export const useSettingsStore = defineStore(STORES.SETTINGS, {
 		isDefaultAuthenticationSaml(): boolean {
 			return this.userManagement.authenticationMethod === UserManagementAuthenticationMethod.Saml;
 		},
+		isWithinUserQuota(): boolean {
+			const userStore = useUsersStore();
+			return (
+				this.userManagement.quota === -1 || this.userManagement.quota > userStore.allUsers.length
```

Comment:
```suggestion
			return (
				this.userManagement.quota === -1 || this.userManagement.quota > useUsersStore().allUsers.length
```

Reply:
If it's equal we don't consider it to be within quota?

Reply:
No, because if user quota === 1 it would evaluate to true, allowing user to invite other users.

Reply:
Still confused by this! If `this.userManagement.quota` is `1` and `userStore.allUsers.length` is `1`, then `this.userManagement.quota === -1 || this.userManagement.quota > userStore.allUsers.length` will resolve to `false || false` and so `isWithinUserQuota()` will return `false`.

But if quota and number of users are equal, then I'd expect `isWithinUserQuota()` to return `true`, because with equal numbers, the instance is at capacity, still properly within quota, and should be prevented from exceeding it.

Reply:
I see. So you see an issue with the name of the method? I've renamed it to `isBelowUserQuota` which should be more representative of what it references. So user would be below the quota(and hence able to invite more users) if quota is `-1` or if it's bigger than current users count. Does that make sense to you?